.storybook/preview.tsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable unicorn/filename-case */
+import React, { FC, PropsWithChildren } from 'react';
+import {
+  ensure as ensureTheme,
+  ThemeProvider as StorybookThemeProvider,
+} from '@storybook/theming';
+import { Decorator, Preview } from '@storybook/react';
+import { ChakraProvider, ChakraProviderProps } from '@chakra-ui/react';
+
+import { breakpoints } from '../src/themes/shared/breakpoints';
+import {
+  autoScout24Theme,
+  autoScoutChakraTheme,
+  motoScout24Theme,
+  motoScoutChakraTheme,
+} from '../src/themes';
+import Fonts from '../src/fonts/Hosted';
+import storybookTheme from './theme';
+
+const themes = {
+  [autoScout24Theme.name]: autoScout24Theme,
+  [motoScout24Theme.name]: motoScout24Theme,
+  [motoScoutChakraTheme.name]: motoScoutChakraTheme,
+  [autoScoutChakraTheme.name]: autoScoutChakraTheme,
+};
+
+const ThemeDecorator: FC<
```

Comment:
Wrapper around the story that ensures the chakra theme is set.
We had that before